packages/plugin-pnp/sources/index.ts

```diff
@@ -27,6 +27,7 @@ export const getPnpPath = (project: Project) => {
   return {
     main: ppath.join(project.cwd, mainFilename as Filename),
     other: ppath.join(project.cwd, otherFilename as Filename),
+    esmLoader: ppath.join(project.cwd, `experimental-pnp-esm-loader.mjs` as Filename),
```

Comment:
I'd go with `.pnp.mjs` - I'd avoid putting it in `.yarn` (cc @nicolo-ribaudo), because some people may have gitignore patterns that would ignore it by default. Having it at the root is also easier for third-party detection.

Reply:
I don't think calling it that is a good idea, people might mistake it for the esm version of `.pnp.cjs` which it isn't